src/os_install.c

```diff
@@ -199,8 +185,26 @@ enum swupd_code install_main(int argc, char **argv)
 	verify_set_option_bundles(cmdline_bundles);
 	verify_set_option_version(cmdline_option_version);
 
-	/* install */
+	/*
+	 * Steps for os-install:
+	 *  1) load_manifests
+	 *  2) download_packs
+	 *  3) extract_packs
+	 *  4) check_files_hash
+	 *  5) validate_fullfiles
+	 *  6) download_fullfiles
+	 *  7) extract_fullfiles (with --download finishes here)
+	 *  8) add_missing_files
+	 *  9) run_postupdate_scripts
+	 */
+	if (cmdline_option_download) {
+		steps_in_os_install = 7;
+	} else {
+		steps_in_os_install = 9;
+	}
 	progress_init_steps("os-install", steps_in_os_install);
+
+	/* install */
```

Comment:
verify?

Reply:
this is actually an install, but we use the verify function for it

Reply:
okay